### ISSUE TITLE
Enclave TRUSTED and COMPLICATED - Reason: CONFIGURATION_NEEDED

### ISSUE THREAD

**Issue by zavalyshyn** [intel/sgx-ra-sample#42]
Hi,

I'm trying to run the sgx-ra-sample on my Intel NUC8i5BEH and I'm getting the `Enclave TRUSTED and COMPLICATED - Reason: CONFIGURATION_NEEDED` error from the server code:

```
---- IAS Advisories --------------------------------------------------------
https://security-center.intel.com

INTEL-SA-00161,INTEL-SA-00219

----------------------------------------------------------------------------

---- IAS Report - JSON - Required Fields -----------------------------------
version               = 3
id:                   = 268214031561656433470273251437907112828
timestamp             = 2020-03-13T09:08:14.917342
isvEnclaveQuoteStatus = CONFIGURATION_NEEDED
isvEnclaveQuoteBody   = AgABAIoLAAAKAAkAAAAAAMd4tdVMkdJ/I6ZyS4g8mxsAAAAAAAAAAAAAAAAAAAAADQ7//wGAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABwAAAAAAAAAHAAAAAAAAAD33e589jj38dgaXhMtuIlGP50V6yHZnXo+9s7SL1gqBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAC9ccY4Dvd8VBfostHOLUtlBLn0GOUEk0JEDP/yRD2VvQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACZwmT1jjtJNIVrh4ynKQH16ux5Kt+PI7Q4NvQt9KcVggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

---- IAS Report - JSON - Optional Fields -----------------------------------
platformInfoBlob  = 1502006500000800000D0D02040180030000000000000000000A00000B000000020000000000000B8AD8426353323B960CC444FEDC763E529066F4882C326C6D5AD52CBEE8E189950423C894317ABA6F487BE04C35936AF225DCB0BFED19E99D4CB822405AD814F380
revocationReason  = 
pseManifestStatus = 
pseManifestHash   = 
nonce             = 
epidPseudonym     = 0H2od+NJ2/hk8kxSocI9XF58K55o859IbA20uBPsi71Sxzjs8znc8Y3dywvgLv8e0fj8BmFkreVlTXe9c433JUytrWjMTi6cQF/CoJEzioQmYTGBF5hVEsXTfjWYFEJR9PMaRoa8RjJ7pJD4J+qWAk/wWrp84DafOiH7OaJVHPo=
----------------------------------------------------------------------------
+++ Verifying report version against API version

---- ISV Enclave Trust Status ----------------------------------------------
Enclave TRUSTED and COMPLICATED - Reason: CONFIGURATION_NEEDED
A Platform Info Blob (PIB) was provided by the IAS
``` 

As suggested in the Intel documentation I have upgraded to the latest BIOS version and disabled hyper-threading via BIOS settings. 

I have Intel SGX SDK/PSW v2.7.1 and Intel SGX Driver v2.6 installed on my system. Although SDK version is not the latest version available, I found out that it's the only one that `sgx-ra-sample` code works with. 

Do you have any suggestions on how to solve this issue?

**Comment by jmechalas** [comment]
Keep checking for an updated BIOS for that NUC. The release cycle of BIOS images from an OEM (even ourselves) often lags these updates.
The SDK version may also be contributing. I am working the updates for 2.9 this week.

**Comment by jmechalas** [comment]
I shoudl add...one of my colleagues is better versed in the TCB updates and system/BIOS configurations, and may have something to add here.

**Comment by zavalyshyn** [comment]
Thanks a lot for clarifying this, John. I'll be checking for BIOS updates then.

**Comment by yyyyyp** [comment]
The recommended value of CPU_SVN increases from 0d0d to 0e0e. Is that reasonable ?  I own six Dell machines. All reports error which indicates that the client is out of date. As far as I know, no  BIOS update were released since 2019.12.

**Comment by yyyyyp** [comment]
> Keep checking for an updated BIOS for that NUC. The release cycle of BIOS images from an OEM (even ourselves) often lags these updates.
> The SDK version may also be contributing. I am working the updates for 2.9 this week.

The recommended value of CPU_SVN increases from 0d0d to 0e0e. Is that reasonable ? I own six Dell machines. All reports error which indicates that the client is out of date. As far as I know, no BIOS update were released since 2019.12.

**Comment by jmechalas** [comment]
At the end of the day, we are at the mercy of our OEMs.
